.bazelrc

```diff
@@ -11,6 +11,7 @@
 startup --host_jvm_args=-Xmx2g
 
 build --workspace_status_command="bash bazel/get_workspace_status"
+build --experimental_local_memory_estimate
```

Comment:
don't put this back

Reply:
Forgot to merge master before I pushed, removed it now